101-bundle-json.md

```diff
@@ -802,7 +802,7 @@ Output specifications are flat (not tree-like), consisting of name/value pairs.
     - `multipleOf`: Output validation requiring that the resulting number be wholly divisible by the number specified. MUST be a number strictly greater than zero. (OPTIONAL)
     - `not`: Output validation requiring that the resulting value NOT match the specified schema. MUST be a JSON schema. (OPTIONAL)
     - `oneOf`: Output validation requiring that the resulting value match ONE of the specified schemas. MUST be a non-empty array of JSON schemas. (OPTIONAL)
-    - `path`: The fully qualified path to a file that will be created (REQUIRED)
+    - `path`: The fully qualified path to a file that will be created (REQUIRED). The path specified MUST be a subpath of /cnab/app/outputs.
```

Comment:
I think the path needs to be a strict subpath of `/cnab/app/outputs`. If the path is equal to `/cnab/app/outputs`, something bad may happen.

Reply:
Yeah, that was the intent. Added _strict_ in front of subpath.